### PR TITLE
fix(v2): docs build

### PR DESCRIPTION
**Pull request by Varixo**
This PR fixes the docs build

### CODE REVIEW

File: packages/qwik/src/optimizer/src/plugins/image-size-runtime.html [hunk 1]
```diff
@@ -311,11 +311,11 @@ <h2 id="title"></h2>
         const browserArea = rect.width * rect.height;
         if (!node.hasAttribute('width') || !node.hasAttribute('height')) {
           skip = true;
-          const computedStyles = node.computedStyleMap();
```
Comment (Varixo) [inline packages/qwik/src/optimizer/src/plugins/image-size-runtime.html:314]: the `computedStyleMap` is not supported in Firefox

**Comment by shairez** [comment]
@Varixo @gioboa maybe it's not showing CF preview because it's done on your fork and not on a local branch on this repo? 🤔🤔

**Comment by Varixo** [comment]
> @Varixo @gioboa maybe it's not showing CF preview because it's done on your fork and not on a local branch on this repo? 🤔🤔

Yeah, maybe

**Comment by Varixo** [comment]
Closing this for now. I will create a new PR with cf deployment

**Pull request closed without merging**